### DISCUSSION TITLE
Does Ariakit v2 support IE 11

### DISCUSSION THREAD

**Discussion by bugzpodder** [ariakit/ariakit#1660] [Questions] [answered]
It seems reakit v1 supports ie 11 [ https://github.com/ariakit/ariakit/issues/648#issuecomment-630407657 ].  Would that change with ariakit?

**Comment by diegohaz** [comment, marked as answer]
It doesn't support IE 11.

**Reply by bugzpodder** [reply]
thank you.  Can you confirm reakit v1 does support ie 11?

**Reply by diegohaz** [reply]
Yep
